README.md

```diff
@@ -132,14 +134,17 @@ Table of Contents
   * https://snap-ci.com/ - Free for public repositories, 1 build at the time
   * http://www.appveyor.com/ - CD service for Windows, free for Open Source
   * [Comparison of Continuous Integration services](https://github.com/ligurio/Continuous-Integration-services)
-  * https://saucelabs.com/ - CI with scalable testing for mobile and web apps, free for Open Source
```

Comment:
That was the proposal of the OP.
Actually it fits in both categories. 
I can leave it as it was, if you want.
Another thing : what do I do with that problematic url? It has nothing to do with this PR.